src/Orchard.Web/Core/Common/Models/CommonPartVersionRecord.cs

```diff
@@ -6,5 +6,6 @@ public class CommonPartVersionRecord : ContentPartVersionRecord {
         public virtual DateTime? CreatedUtc { get; set; }
         public virtual DateTime? PublishedUtc { get; set; }
         public virtual DateTime? ModifiedUtc { get; set; }
+        public virtual string ModifiedBy { get; set; }

```

Comment:
To be complete this PR could use the ModifiedBy somewhere in the app, maybe in the admin summary, in place of the current Owner. Don't proceed until we get some feedback from other committers.